crates/holochain/src/core/workflow/countersigning_workflow/refresh.rs

```diff
@@ -67,8 +67,7 @@ pub async fn refresh_workspace_state(
                     .write_async({
                         let agent = agent.clone();
                         move |txn| -> SourceChainResult<(CurrentCountersigningSessionOpt, bool)> {
-                            let maybe_current_session =
-                                current_countersigning_session(txn, Arc::new(agent.clone()))?;
+                            let maybe_current_session = current_countersigning_session(txn)?;
```

Comment:
Again, and again, we have to update code that isn't usable :( Even if it doesn't take long, there is a cost to this